content/en/docs/instrumentation/js/instrumentation.md

```diff
@@ -25,29 +27,32 @@ function doWork() {
 
 ## Initializing a Tracer
 
-As you have learned in the previous [Getting Started](../getting-started) guide you
-need a TracerProvider and an Exporter. Install the dependencies and add them to
-the head of your application code to get started:
+As you have learned in the previous [Getting Started](../getting-started) guide
+you need a TracerProvider and an Exporter. Install the dependencies and add them
+to the head of your application code to get started:
 
 ```shell
 npm install @opentelemetry/api
 npm install @opentelemetry/sdk-trace-base
 ```
 
-Next, initialize a tracer, preferably in a separate file (e.g., `instrumentation-setup.js`):
+Next, initialize a tracer, preferably in a separate file (e.g.,
+`instrumentation-setup.js`):
 
 ```javascript
 const {
   BasicTracerProvider,
   ConsoleSpanExporter,
-  SimpleSpanProcessor,
+  BatchSpanProcessor,
```

Comment:
Batch span processor makes sense in some cases (typical server app) but not in others (serverless). It might make sense to add a comment about this because a serverless container may be killed before a batch is exported.